protocol/0028-GOVE-governance.md

```diff
@@ -355,7 +355,37 @@ All in memory active governance transfers must be included in the snapshot of th
 4. Enacted governance transfers are therefore available to be queried via the regular transfer API in data node.
 5. Governance initiated transfers are subject to neither minimum transfer amounts nor to fees.
 
-## 6. Freeform governance proposal
+## 6. Change market state
+
+A governance proposal to change the state of the market.
+Multiple concurrent proposals are allowed. If more than one gets successfully voted in the one which arrived last gets used. If a proposal has already been successful then additional market state change proposals are still allowed as long as their enactment date is no later then that of the last successful vote.
+
+Refer to subsections below for allowed state changes.
+
+### 6.1. Move market to a closed state
+
+Any type of market (either fixed expiry market or perpetual) can be closed via a governance vote.
+
+A proposal to close a market contains:
+
+1. an enactment time
+1. final settlement price (not required for spot markets)
+
+Once market is closed the process cannot be reversed.
```

Comment:
Do the market change proposal voting rules apply? I would suggest they do. 

Btw. on that we have that only an LP with a certian ELS can make the proposal (min tokens *and* min ELS met). I think we should change those to (min tokens *OR* min ELS met).

Reply:
Clarified now, happy to add the modification you mentioned, but perhaps in a separate PR?